Library/Homebrew/test/utils/git_spec.rb

```diff
@@ -0,0 +1,181 @@
+require "utils/git"
+
+describe Git do
+  before(:all) do
+    git = HOMEBREW_SHIMS_PATH/"scm/git"
+    file = "lib/blah.rb"
+    repo = Pathname.new("repo")
+    FileUtils.mkpath("repo/lib")
+    system "#{git} init"
+    FileUtils.touch("repo/#{file}")
+    File.open(repo.join("file").to_s, "w") { |f| f.write("blah") }
+    system "#{git} add repo/#{file}"
```

Comment:
Any specific reasons for preferring `system "git", "add", "repo/#{file}"` over ` system "#{git} add repo/#{file}"` ?